### PR TITLE
Authority change

### PR DESCRIPTION
**Pull request by sukrit-ym**

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -8,5 +8,5 @@
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
     <color name="ymAccentBlue">#FF03DAC5</color>
-    <color name="ym_icon_color">#FFFFFF</color>
```
Comment (sankalp211) [inline app/src/main/res/values/colors.xml:11]: Why have we commented the icon color?

File: YMChat/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -7,7 +7,6 @@
     <string name="ym_lite_chatbot_base_url">file:///android_asset/yellow-index-lite.html</string>
     <string name="ym_fab_voice_button">voice button</string>
     <string name="application_id_for_provider">com.yellowmessenger.ymchatexample.fileprovider</string>
-    <string name="ym_file_provider">@string/application_id_for_provider</string>
```
Comment (sankalp211) [inline YMChat/src/main/res/values/strings.xml:10]: We decided to keep this string. Was that something else?

**Review by sankalp211 — CHANGES_REQUESTED** [review]

File: YMChat/src/main/res/values/strings.xml (hunk 2 above)
Comment (sukrit-ym) [reply YMChat/src/main/res/values/strings.xml:10]: that was above one..this we use internally

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (sukrit-ym) [reply app/src/main/res/values/colors.xml:11]: Not required.. this is an example how once can override

File: YMChat/src/main/res/values/strings.xml (hunk 2 above)
Comment (sukrit-ym) [reply YMChat/src/main/res/values/strings.xml:10]: that was above one..this we used internally

**Review by sankalp211 — APPROVED** [review]

**Pull request merged**